x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/result_settings_text_fields_header.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiTableHeader, EuiTableHeaderCell, EuiToolTip, EuiIcon } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+export const ResultSettingsTextFieldsHeader: React.FC = () => {
+  return (
+    <>
+      <EuiTableHeader className="resultSettingsTable__columnLabels">
+        <EuiTableHeaderCell align="left" />
+        <EuiTableHeaderCell align="center" colSpan={2}>
+          {i18n.translate('xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawTitle', {
+            defaultMessage: 'Raw',
+          })}
+          <EuiToolTip
+            position="top"
+            content={i18n.translate(
+              'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawDescription',
+              {
+                defaultMessage:
+                  'A raw field is an exact representation of a field value. Must be at least 20 characters. Defaults to the entire field.',
+              }
+            )}
+          >
+            <EuiIcon type="questionInCircle" />
```

Comment:
Not a change request, just an accessibility note for whenever we refactor this UI later - this should likely be an `EuiButtonIcon` to allow screen readers & keyboard users to interact w/ this tooltip

Reply:
Oh interesting. I would have just assumed that `EuiToolTip` surrounding it was already accessible. I'll drop a TODO in here, so we don't lose sight of it when we clean up this UI.

Reply:
I ended up going through and adding inline TODOs for all of your optional feedback. Not sure if that helps or not... ed490af326b9cf6f6cec9b9d9662b8f298fba8de. Happy to revert if that's just noise.

I also might take a stab at just cleaning these up quick. It will probably only take a moment, and then if for some reason we don't get back around to cleaning up this UI, at least things will be right.

Reply:
I'm good with whatever! Appreciate TODOs as well. I actually assumed the same for EuiTooltip but it looks like they do have a note about accessibility/focus on their docs: https://elastic.github.io/eui/#/display/tooltip

and it looks like `EuiIconTip` also has some extra work on their part to add focus/tabindexing. Pretty neat!

Reply:
Ah man, I was all excited to tell you that I found EuiIconTip but you beat me to it: https://github.com/elastic/kibana/pull/95462/commits/b8cc5c5f34c5c3dc98d424592a9af9422ccb1b84

Reply:
HAHA I had totally forgetten about it in my original comment for what it's worth! This is awesome!